addons/binding/org.openhab.binding.onebusaway/src/main/java/org/openhab/binding/onebusaway/config/StopConfiguration.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
These were generated this way, and I just copied the generated headers.  Probably worth making sure the generation script does the right thing (and maybe not hardcode the date so it works in the future without updating all the places since this seems to be a common problem).

Regardless, I'll fix this in all the files I've added.